specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json

```diff
@@ -39,7 +39,7 @@
         "tags": [ "v1consumer" ],
         "summary": "Consume records using a simple consumer",
         "description": "Consume records from one partition of a topic beginning with a specific offset. By default count is 1. It can read maximum of 6 megabytes of data at once",
-        "operationId": "consumeTopicInPartitionWithOffset",
+        "operationId": "Consumer_ConsumeTopicInPartitionWithOffset",
```

Comment:
Please help me fix the Prettier failure. I am not sure what is incorrect and how to fix it.